DirectProgramming/C++SYCL_FPGA/Tutorials/Features/ip_authoring_interfaces_overview/mm-host/Readme.md

```diff
@@ -0,0 +1,181 @@
+# Memory-Mapped Host Interfaces
+This implementation uses a register-mapped invocation interface, and demonstrates how to use `annotated_arg` to customize the memory-mapped host data interface.
+
+![](../assets/ddr.svg)
+
+## Invocation Interface
+By default, an un-decorated oneAPI kernel will have all its control signals and arguments mapped into the IP component's control/status register (CSR).
+
+## Data Interface - MM Host
+The pointer arguments `A_in`, `B_in`, `C_out` and scalar argument `len` are passed through the IP component's CSR. In this example design, the memory-mapped host interfaces pointed to by `A_in`, `B_in`, `C_out` are customized using `annotated_arg`.
+
+You can customize memory-mapped interfaces of your IP component if the component uses a unified shared memory (USM) host pointer to access data. To customize the interface, declare your pointer arguments with the templated type `annotated_arg`.
+
+### Declare `annotated_arg``
+Explicit MM Host should be declared as functor member of your kernel:
+
+```cpp
+  sycl::ext::oneapi::experimental::annotated_arg<type, properties>
```

Comment:
i think this snippet may be a little redundant since you show the full syntax below. maybe you can show it being used in a functor here?